files/system/river/etc/river/init

```diff
@@ -170,13 +170,11 @@ riverctl map normal Super Escape spawn "swaylock -f"
 riverctl map normal None Print spawn 'grim -g "$(slurp -d)" - | wl-copy'
 
 riverctl spawn dunst
-riverctl spawn pipewire
-riverctl spawn pipewire-pulse
-riverctl spawn wireplumber
 riverctl spawn "/usr/libexec/xfce-polkit"
 riverctl spawn waybar
 riverctl spawn kanshi
 riverctl spawn nm-applet
+riverctl spawn "swaybg --image /usr/share/backgrounds/f41/default/f41-01-night.png"
```

Comment:
should be `/usr/share/backgrounds/default.png` I believe

Reply:
ok, will fix

Reply:
it does show /usr/share/backgrounds/default.png on a fresh user, resolving conversation